test_elasticsearch_dsl/test_integration/test_update_by_query.py

```diff
@@ -0,0 +1,35 @@
+from elasticsearch_dsl.update_by_query import UpdateByQuery
+from elasticsearch_dsl.search import Q
+
+def test_update_by_query_no_script(data_client):
+    ubq = UpdateByQuery(using=data_client).index('git').filter(~Q('exists', field='is_public'))
+    response = ubq.execute()
+
+    assert response.total == 52
+    assert response['took'] > 0
+    assert not response.timed_out
+    assert response.updated == 52
+    assert response.deleted == 0
+    assert response.took > 0
+
+def test_update_by_query_with_script(data_client):
+    ubq = UpdateByQuery(using=data_client).index('git')\
+        .filter(~Q('exists', field='parent_shas'))\
+        .script(source='ctx._source.is_public = false')
+    ubq = ubq.params(conflicts='proceed')
+
+    response = ubq.execute()
+    assert response.total == 2
+    assert response.updated == 1
+    assert response.version_conflicts == 1
+
+def test_delete_by_query_with_script(data_client):
```

Comment:
This looks like a dangerous test - `data_client` uses the `git` and `flat-git` indices in a read-only fashion and it is therefore not torn down after every single test. Altering any data in those indices will have consequences in other tests and is to be avoided. Please use `write_client` and an index starting with `test-` for any test that alters documents in elasticsearch.